lib/plugins/create/templates/aws-golang/handler.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const spawn = require('child_process').spawn;
+
+// Your first function handler
+module.exports.hello = (event, context, cb) => {
+  const proc = spawn('./hello', [JSON.stringify(event)]);
+  let eventData;
+  let err;
+
+  proc.stdout.on('data', (data) => {
+    eventData = JSON.parse(data.toString('ascii'));
+    if (typeof eventData === 'string') {
+      eventData = JSON.parse(eventData);
+    }
+  });
+
+  proc.stderr.on('data', (data) => {
+    err = data.toString('ascii');
+  });
+
+  proc.on('close', (code) => {
+    if (code !== 0) {
+      cb(new Error('Process exited with non-zero status code'));
+    } else if (err) {
+      cb(new Error(err));
+    } else {
+      cb(null, { message: 'Go Serverless v1.0! Your function executed successfully!', eventData });
```

Comment:
@flomotlik  This is meant to copy what is currently in the Nodejs template: https://github.com/serverless/serverless/blob/v1.0/lib/plugins/create/templates/aws-nodejs/handler.js#L5